js/bitfinex.js

```diff
@@ -678,6 +676,8 @@ module.exports = class bitfinex extends Exchange {
             'limit': limit,
             'start': since,
         };
+        if (typeof since === 'undefined')
```

Comment:
There's very little point in adding it to the request above and then removing that param from the request. Instead we should do
```
let request = {
    'limit': limit,
};
if (typeof since !== 'undefined')
    request['start'] = since;
```

However, this edit can break the querying range for bitfinex... But, no worries, I'll fix it properly and will let you know.

Reply:
tanks

Reply:
@marcoracer thx for reporting back!